content/en/real_user_monitoring/connect_rum_and_traces/_index.md

```diff
@@ -209,7 +213,36 @@ RUM supports several propagator types to connect resources with backends that ar
       - `tracecontext`: [W3C Trace Context](https://www.w3.org/TR/trace-context/) (`traceparent`)
       - `b3`: [B3 single header](https://github.com/openzipkin/b3-propagation#single-header) (`b3`)
       - `b3multi`: [B3 multiple headers](https://github.com/openzipkin/b3-propagation#multiple-headers) (`X-B3-*`)
+{{% /tab %}}
+
+{{% tab "iOS RUM" %}}
+1. Set up RUM to connect with APM as described above.
```

Comment:
```suggestion
1. [Set up RUM to connect with APM](#setup-rum).
```

Reply:
Is that anchor link linking to the correct section?

Reply:
If we change it here we should also change in the Browser section to make it coherent, no?

Reply:
Oh yeah, okay, let's just leave it for now.